.github/workflows/build.yml

```
@@ -177,21 +177,21 @@ jobs:
           if [[ ${{ matrix.os }} == windows-* ]]; then
             curl -L "${{ matrix.cudnn_url }}" > download/cudnn.zip
   
-            unzip download/cudnn.zip cuda/bin/*.dll -d download/cudnn_tmp
+            unzip download/cudnn.zip cudnn-*/bin/*.dll -d download/cudnn_tmp
   
             mkdir -p download/cudnn/bin
-            mv download/cudnn_tmp/cuda/bin/*.dll download/cudnn/bin/
+            mv download/cudnn_tmp/cudnn-*/bin/*.dll download/cudnn/bin/
             rm -rf download/cudnn_tmp
   
             rm download/cudnn.zip
           else
             curl -L "${{ matrix.cudnn_url }}" > download/cudnn.tgz
 
-            tar -zxf download/cudnn.tgz -C download/
+            tar -Jxf download/cudnn.tgz -C download/
```

```suggestion
            curl -L "${{ matrix.cudnn_url }}" > download/cudnn.tar.xz

            tar -Jxf download/cudnn.tar.xz -C download/
```

Linux向けのcudnnの配布形式が`tgz`から`tar.xz`に変わったということのようなので、ここも`tar.xz`にしておくとよさそうです！

---

確かにそうですね、ご指摘ありがとうございます！　修正しました。